modules/security-context-constraints-command-reference.adoc

```diff
@@ -93,24 +93,21 @@ the default SCCs.
 //other than priority, users, groups, labels, and annotations.
 ====
 
-[id="deleting-security-context-constraints_{context}"]
-== Deleting an SCC
+[id="updating-deleting-security-context-constraints_{context}"]
+== Updating and deleting an SCC
+
+[WARNING]
+====
+Please avoid updates and removals of the default SCCs. These will be regenerated
```

Comment:
```suggestion
Do not remove the default SCCs. They are recreated
```

If you don't want them to do it, then you need to say so. "Please" is not allowed by our style guide.